demo/api/schema.gql

```diff
@@ -607,7 +612,7 @@ type Query {
   userPermissionsPermission(id: ID!, userId: String): UserPermission!
   userPermissionsAvailablePermissions: [String!]!
   userPermissionsContentScopes(userId: String!, skipManual: Boolean): [JSONObject!]!
-  userPermissionsAvailableContentScopes: [JSONObject!]!
+  userPermissionsAvailableContentScopes: [AvailableContentScope!]!
```

Comment:
Why is this only used for `userPermissionsAvailableContentScopes`, but not for `userPermissionsContentScopes`? Could there be other queries/fields where the label could also be relevant?